### DISCUSSION TITLE
Request: Integration For Exotic Birds

### DISCUSSION THREAD

**Discussion by SonicTheHedgehog58987276** [vectorwing/FarmersDelight#863] [Ideas]
### Minecraft version

1.20.1

### Farmer's Delight version

1.2.3

### Forge version

47.2.4

### Description

it would be nice if the mod could make it so i can extract more feathers from birds

### Steps to reproduce

no steps (only a request)

### Mod list

only a request for integration

### Minimal instance

- [ ] I have tested this on a minimal instance

### Performance and shader mods

- [ ] I am using performance or shader mods

**Comment by vectorwing** [comment]
You can add integration with Exotic Birds's animals through datapacks. FD uses Forge's loot modifier system to add more drops when killing with knives, and those can be extended to include a modifier for any modded mob.

I'm considering some improvements to this compatibility in the future, but it may take a while for me to get to it.

**Comment by SonicTheHedgehog58987276** [comment]
thanks

**Comment by SonicTheHedgehog58987276** [comment]
where is the data pack?
